web/src/components/Feedback.tsx

```diff
@@ -87,28 +111,72 @@ const Feedback = ({
   return (
     <Container fullWidth={isSearchFeedback}>
       {isSearchFeedback ? (
-        <InputSection id='searchTerm' title={t('searchTermDescription')}>
-          <Input id='searchTerm' value={searchTerm} onChange={setSearchTerm} />
-        </InputSection>
+        <FormControl error={submitted && !searchTerm} variant='outlined'>
+          <FormHelperText id='searchTermDescription'>
+            {submitted && !searchTerm ? t('noteFillFeedback') : t('searchTermDescription')}
+          </FormHelperText>
+          <InputLabel htmlFor='searchTerm' />
+          <OutlinedInput
+            id='searchTerm'
+            value={searchTerm}
+            aria-describedby='searchTermDescription'
+            onChange={e => setSearchTerm(e.target.value)}
+            fullWidth
+            required
+          />
+        </FormControl>
       ) : (
-        setRating && <FeedbackButtons rating={rating} setRating={setRating} />
+        <FormControl error={submitted && rating === null}>
+          <FeedbackButtons rating={rating} setRating={setRating} />
+          {submitted && rating === null && <FormHelperText>{t('noteFillFeedback')}</FormHelperText>}
+        </FormControl>
       )}
-
-      <InputSection
-        id='comment'
-        title={t(commentTitle)}
-        description={t('commentDescription', { appName: buildConfig().appName })}
-        showOptional>
-        <Input id='comment' value={comment} onChange={onCommentChanged} rows={DEFAULT_ROWS_NUMBER} />
-      </InputSection>
-
-      <InputSection id='email' title={t('contactMailAddress')} showOptional>
-        <Input id='email' value={contactMail} onChange={onContactMailChanged} />
-      </InputSection>
-      <PrivacyCheckbox language={language} checked={privacyPolicyAccepted} setChecked={setPrivacyPolicyAccepted} />
-      {submitFeedbackDisabled && <Note text={t(feedbackFilled ? 'noteFillFeedback' : 'notePrivacyPolicy')} />}
+      <OptionalHint>({t('common:optional')})</OptionalHint>
+      <FormControl fullWidth variant='outlined'>
+        <InputLabel htmlFor='comment'>{t(commentTitle)}</InputLabel>
+        <OutlinedInput
+          id='comment'
+          value={comment}
+          onChange={e => onCommentChanged(e.target.value)}
+          aria-describedby='commentDescription'
+          label={t(commentTitle)}
+          multiline
+          rows={4}
+        />
+        <FormHelperText id='commentDescription'>
+          {t('commentDescription', { appName: buildConfig().appName })}
+        </FormHelperText>
+      </FormControl>
+      <OptionalHint>({t('common:optional')})</OptionalHint>
```

Comment:
❓ Is there any reason why we don't use the `required` prop from material text fields instead?
https://mui.com/material-ui/react-text-field/#form-props

Reply:
do you mean optional text?

Reply:
I also agree with you, but @hauf-toni suggested that it would be better to have an extra hint for not required fields 
> Thank you for your questions, here are my thoughts on this:
> 💬 we should definitely mark these entries as optional so that users are less hesitant to actually send feedback as they will be informed that they do not have to fill in these fields. Thanks for adding them to the ui.

Reply:
I think there is no other way to add `optional` except for adding an extra div above the optional fields